draft-keytrans-mcmillion-protocol.md

```diff
@@ -499,7 +497,8 @@ The leaf nodes of a prefix tree are serialized as:
 
 ~~~ tls
 struct {
-    opaque key_version<VRF.Nh>;
+  opaque value<Hash.Nh>
```

Comment:
I don't think it makes sense to store the `value` in the leaf node, as the value can be immediately computed from `key_version`. So it is redundant when this struct is used later on.